lib/collnew/src/lib.rs

```diff
@@ -0,0 +1,36 @@
+#[macro_export]
```

Comment:
Would it make sense to put this into a new "vector-core" or "vector-shared" crate?

Reply:
"vector-core"? I think "vector-shared" is a good option, at least there will be fewer packages.

Reply:
Moved to `shared`.